posts/2021-11-09-now-with-more-elixir.md

```diff
@@ -0,0 +1,208 @@
+       for section <- Keyword.keys(config) do
+       Helpers.lesson_url(Endpoint, :index, section, locale)
+       end
+
+     section_indexes ++ translated_lesson_links
+   end
+
+   defp translated_lesson?(section, lesson, locale) do
+     case Lessons.get(section, lesson, locale) do
+     {:ok, _} -> true
+     _ -> false
+     end
+   end
+
+   defp supported_locales do
+     :school_house
+     |> Application.get_env(SchoolHouseWeb.Gettext)
+     |> Keyword.get(:locales)
+   end
+   ```
+
+Both tasks are run in our release process and stored in the static directory to be served alongside things like `robots.txt`:
+
+```elixir
+plug Plug.Static,
+  at: "/",
+  from: :school_house,
+  gzip: false,
+  only: ~w(css fonts images js favicon.ico robots.txt feed.xml sitemap.xml)
+```
+
+### Roadmap Forward
+
+#### New Content
+
+With this launch we also restructured our content into new sections! This paves the way for much more content and a more managable way to organize and consume it. Some of the new content you can be on the look out for:
+
+1. **Advanced Content**
+
+   1. Dedicated meta programming lesson that explores this powerful functionality
+   2. Greatly expanding on our Specifications and Type content
+
+2. **Data Processing**
+
+   1. New Leson: Broadway library by Dashbit
+   2. New Lesson: Flow
+
+   Together these give us coverage of the data trifect: GenStage, Flow, and Broadway.
+
+3. **Ecto**
+
+   1. Expanding existing Ecto content
+   2. New Lesson: Advanced Ecto querying techniques
+
+4. **Storage**
+
+   1. New Lesson: Redix library
+   2. New Lesson: Cachex library
+
+5. **Fundamentals**
+
+   1. New Lesson: Functional Programming 101
+   2. New Lesson: Functional Programming 102
+   3. New Lesson: Data structures
+
+We continue to discuss the possibility and value of expanding into Phoenix and Erlang content more heavily. We'd love your feedback! Interested in contributing to some of these lessons? Have suggestions for content? Don't hesitate to reach out or get involved!
+
+Our lessons aren't all! We've got a lot of exciting blog content in mind for the coming months:
+
+1. We plan to resume our book, course, and conference reviews to continue to provide the community with an unbiased resource on which learning materials available to them are worth the expense.
+2. In conjunction with Elixir Companies we're going to be kicking off a new blog series on companies using Elixir. Some of the questions you've been asking we aim to answer: Who's using Elixir? How did they end up selecting Elixir? How has the onboarding been? What is Elixir used for?
+3. Community Spotlights! There's **alot** of amazing people in the community many of which don't get the recoginition they deserve. We want to fix that by selecting and highlighting those who're giving back to the community and moving us all forward. There's some terrific people involved in Elixir overshadowed by a smaller louder group and we want to change that!
```

Comment:
`a lot` is two words, right? :)

Reply:
To add to this, I believe it should be `There are **a lot** of amazing people...`

Reply:
And then the last sentence it should be `There are some terrific...`

Reply:
```suggestion
3. Community Spotlights! There are **a lot** of amazing people in the community many of which don't get the recoginition they deserve. We want to fix that by selecting and highlighting those who're giving back to the community and moving us all forward. There are some terrific people involved in Elixir overshadowed by a smaller louder group and we want to change that!
```